src/bin/pg_autoctl/state.h

```diff
@@ -63,6 +63,10 @@ typedef enum
  */
 #define state_matches(x, y) (x == ANY_STATE || y == ANY_STATE || x == y)
 
+#define IS_STANDBY(state) (state_matches((state), CATCHINGUP_STATE) || \
+						   state_matches((state), SECONDARY_STATE) || \
+						   state_matches((state), MAINTENANCE_STATE))
+
```

Comment:
As mentioned before, I don't think we need that. If we do though, we have more states that belong to a standby...